docs/index.rst

```diff
@@ -0,0 +1,28 @@
+TorchServe
+==================================
+
+TorchServe is a flexible and easy to use tool for serving PyTorch models.
+
+.. warning ::
+     TorchServe is experimental and subject to change.
+
+
+Basic Features
+----------------
+
+* `Serving Quick Start <https://pytorch.org/serve/install.html#serve-a-model>`_ - Basic server usage tutorial
+* `Model Archive Quick Start <https://pytorch.org/serve/model-archiver.html#creating-a-model-archive>`_ - Tutorial that shows you how to package a model archive file.
+* `Installation <https://pytorch.org/serve/install.html##install-torchserve>`_ - Installation procedures
+* `Serving Models <https://pytorch.org/serve/server.md>`_ - Explains how to use `torchserve`.
```

Comment:
All of these could probably be .html instead of .md

Reply:
Updated to relative link that references the .md file